pygmt/src/xyz2grd.py

```diff
@@ -40,9 +46,94 @@ def xyz2grd(table, **kwargs):
+        *title* (anything you like), and *remark* (anything you like). Items
+        not listed will remain untouched. Give a blank name to completely reset
+        a particular string. Use quotes to group texts with more than one word.
+        If any of your text contains plus symbols you need to escape them
+        (place a backslash before each plus-sign) so they are not confused with
+        the option modifiers.  Alternatively, you can place the entire
+        double-quoted string inside single quotes.
     {I}
     {R}
     {V}
+    onecolumn : str
+        [*flags*]
+        Read a 1-column ASCII [or binary] table. This assumes that all the
+        nodes are present and sorted according to specified ordering
+        convention contained in *flags*. If incoming data represents rows,
+        make *flags* start with **T**\ (op) if first row is y
+        = ymax or **B**\ (ottom) if first row is y = ymin.
+        Then, append **L** or **R** to indicate that first element is at
+        left or right end of row. Likewise for column formats: start with
+        **L** or **R** to position first column, and then append **T** or
+        **B** to position first element in a row. **Note**: These two
+        row/column indicators are only required for grids; for other tables
+        they do not apply. For gridline registered grids: If data are periodic
+        in x but the incoming data do not contain the (redundant) column at
+        x = xmax, append **x**. For data periodic in y without redundant row at
+        y = ymax, append **y**. Append **s**\ *n* to skip the first *n* number
+        of bytes (probably a header). If the byte-order or the words needs
+        to be swapped, append **w**. Select one of several data types (all
+        binary except **a**):
+
+        **A** ASCII representation of one or more floating point values per
+        record
+
+        **a** ASCII representation of a single item per record
+
+        **c** int8_t, signed 1-byte character
+
+        **u** uint8_t, unsigned 1-byte character
+
+        **h** int16_t, signed 2-byte integer
+
+        **H** uint16_t, unsigned 2-byte integer
+
+        **i** int32_t, signed 4-byte integer
+
+        **I** uint32_t, unsigned 4-byte integer
+
+        **l** int64_t, long (8-byte) integer
+
+        **L** uint64_t, unsigned long (8-byte) integer
+
+        **f** 4-byte floating point single precision
+
+        **d** 8-byte floating point double precision
+
+        Default format is scanline orientation of ASCII numbers: **-ZTLa**.
+        The difference between **A** and **a** is that the latter can decode
+        both *date*\ **T**\ *clock* and *ddd:mm:ss[.xx]* formats but expects
+        each input record to have a single value, while the former can handle
+        multiple values per record but can only parse regular floating point
+        values. Translate incoming *z*-values via the **-i**\ 0 option and
+        needed modifiers.
```

Comment:
I don't think I have ever actually seen a data file that contains x,y,z values in a 1-column ASCII/binary table. Has anyone else? I wonder if this is a relic of some older data standards, in which case it might not be worth including in PyGMT since it's such an obscure argument and would be quite difficult to make pythonic.

Reply:
> a data file that contains x,y,z values in a 1-column ASCII/binary table.

I think it means the 1-column ASCII/binary file only contains the Z values. The X and Y coordinates are determined from `-R` and `-I` option automatically, because the data are uniformly spaced in X and Y dimensions.

Reply:
Ah, I misunderstood the purpose. Yes, I agree that is quite common and should be included.